src/styles/index.scss

```diff
@@ -1,3 +1,152 @@
-body {
+.body {
   margin: 0;
 }
+
+.button {
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  margin: 16px;
+  background: #00acdc;
+  border: 0;
+  align-items: center;
+  color: #fff;
+  font-family: Roboto, sans-serif;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+
+  &:hover {
+    background: #fff;
+    border: 1px solid #00acdc;
+    color: #00acdc;
+  }
```

Comment:
The `.button` class has multiple hard-coded values (width, height, margin, background, etc.). Consider using variables for colors and possibly for sizes if they are reused across the project. This will improve maintainability and consistency of the code. Refer to the checklist item: 'Use variables and avoid magic numbers in your code.'